obspy/sac/core.py

```diff
@@ -256,8 +403,9 @@ def writeSAC(stream, filename, byteorder="<", **kwargs):  # @UnusedVariable
 
     :type stream: :class:`~obspy.core.stream.Stream`
     :param stream: The ObsPy Stream object to write.
-    :type filename: str
-    :param filename: Name of file to write.
+    :type filename: str or file-like object
+    :param filename: Name of file to write. If it is not a filename, it only
```

Comment:
There are two 'it's in this sentence, but they do not refer to the same thing. Should try and avoid that.